package.json

```diff
@@ -99,7 +99,7 @@
     "react": "17.0.1",
     "react-autobind": "^1.0.6",
     "react-dom": "^15.4.2",
-    "react-native": "0.64.1",
+    "react-native": ">=0.65.0",
```

Comment:
This should be reverted, migrating to higher version of RN requires more than just changing the version in package.json

Reply:
Didn’t know that, sorry about that. 
I’ll fix.